packages/core/test/discovery.test.js

```diff
@@ -871,6 +871,58 @@ describe('Discovery', () => {
       ]));
     });
 
+    it('captures fonts with valid auth credentials', async () => {
+      percy.loglevel('debug');
+
+      const fontAuthDOM = dedent`
+       <html>
+       <head>
+         <style>
+          @font-face { font-family: "test"; src: url("font-auth/font.woff") format("woff"); }
+          body { font-family: "test", "sans-serif"; }
+         </style>
+       </head>
+       <body>
+         <p>Hello Percy!<p>
+         ${' '.repeat(1000)}
+       </body>
+       </html>
```

Comment:
Very small nitpick, these all look like they could be indented one more space.

Right now it's indented 7 spaces, when you probably intended it to be an even 8.

Reply:
![Nothing gets past this guy](https://y.yarn.co/e20692f0-4923-405a-9b94-13429aee4ddc_text.gif)